### PR TITLE
build: fix Docker builds

### LINKED ISSUE #87: Unable to build Dockerfile
```
Package qt5-default is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package python3.9-dev
E: Couldn't find any package by glob 'python3.9-dev'
E: Couldn't find any package by regex 'python3.9-dev'
E: Unable to locate package python3.9-distutils
E: Couldn't find any package by glob 'python3.9-distutils'
E: Couldn't find any package by regex 'python3.9-distutils'
E: Unable to locate package enchant
E: Package 'qt5-default' has no installation candidate
```

Then, I tried ubuntu:18.04 cause I saw somewhere qt5-default wasn't on Debian bullseye:
```
Reading state information...
E: Unable to locate package python3.9
E: Couldn't find any package by glob 'python3.9'
E: Couldn't find any package by regex 'python3.9'
E: Unable to locate package python3.9-dev
E: Couldn't find any package by glob 'python3.9-dev'
E: Couldn't find any package by regex 'python3.9-dev'
E: Unable to locate package python3.9-distutils
E: Couldn't find any package by glob 'python3.9-distutils'
E: Couldn't find any package by regex 'python3.9-distutils'
```

### PR DESCRIPTION
**Pull request by rr-**
Resolves #87

### CODE REVIEW

**Comment by rr-** [comment]
@pea-ppod could you please verify if this works for you?

File: .github/workflows/bubblesub.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: bubblesub
 
-on: [push, pull_request]
+on: [pull_request]
```
Comment (Luni-4) [inline .github/workflows/bubblesub.yml:3]: Why have we removed the `push` event?

File: Dockerfile [hunk 2]
```diff
@@ -10,7 +10,7 @@ ENV MPV_PACKAGES="\
 ENV FFMPEG_PACKAGES="\
     libavcodec-dev libavformat-dev libavdevice-dev zlib1g-dev"
 ENV BUBBLESUB_PACKAGES="\
-    python3.9 python3.9-dev python3.9-distutils enchant libfftw3-dev xvfb qt5-default"
```
Comment (Luni-4) [inline Dockerfile:13]: Why isn't `enchant` package more necessary? What's different?

**Review by Luni-4 — CHANGES_REQUESTED** [review]
Seems good to me, just small changes and clarifications

File: .github/workflows/bubblesub.yml (hunk 1 above)
Comment (rr-) [reply .github/workflows/bubblesub.yml:3]: IMO `pull_request` is enough, no need to run the jobs twice.

File: Dockerfile (hunk 2 above)
Comment (rr-) [reply Dockerfile:13]: - The setup is broken (the packages ceased to exist) and I haven't had the time to debug
- We never test it anyway
- It's "optional" e.g. only used by the spell checker and the like

File: .github/workflows/bubblesub.yml (hunk 1 above)
Comment (Luni-4) [reply .github/workflows/bubblesub.yml:3]: The main problem is when we decide to push directly on `master`, in that case any CI check will be triggered. I don't agree with removing this event

File: Dockerfile (hunk 2 above)
Comment (Luni-4) [reply Dockerfile:13]: I see, we can remove it then

File: .github/workflows/bubblesub.yml (hunk 1 above)
Comment (rr-) [reply .github/workflows/bubblesub.yml:3]: In that case leaving it as `push` is enough since it's also triggered for PRs.

**Review by Luni-4 — APPROVED** [review]

**Pull request merged**